### PR TITLE
BaSP-TG-203: Update shared component Table

### PR DESCRIPTION
**Pull request by LucasMattia**

### CODE REVIEW

**Review by gregorioelias — APPROVED** [review]

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ const Admins = (props) => {
     dispatch(getAdmin());
   }, []);
 
+  console.log(dataResponse, 'hola');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/Admins/index.js:21]: We need this console.log?

File: src/Components/Projects/index.js [hunk 2]
```diff
@@ -18,8 +18,11 @@ const Projects = () => {
 
   useEffect(async () => {
     dispatch(getProject());
+    console.log(dataResponse, 'hola2');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/Projects/index.js:21]: We need this console.log?

File: src/Components/Projects/index.js [hunk 3]
```diff
@@ -18,8 +18,11 @@ const Projects = () => {
...
   }, []);
 
+  console.log(dataResponse, 'hola');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/Projects/index.js:24]: We need this console.log?

File: src/Components/redux/modules/projects/thunks.js [hunk 4]
```diff
@@ -16,29 +16,30 @@ import {
...
+//     }
+//     changedDate = `${month}-${day}-${year}`;
+//   }
+//   return changedDate;
+// };
 export const getProject = () => {
   return async (dispatch) => {
     dispatch(getProjectPending());
     const token = sessionStorage.getItem('token');
+    console.log(token, 'token');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/redux/modules/projects/thunks.js:42]: We need this console.log?

File: src/Components/redux/modules/projects/thunks.js [hunk 5]
```diff
@@ -49,14 +50,9 @@ export const getProject = () => {
...
       });
       const data = await response.json();
-      data.data.map((project) => {
-        project.active = project.active ? 'true' : 'false';
-        project.createdAt = changeDate(project.createdAt);
-        project.startDate = changeDate(project.startDate);
-        project.endDate = changeDate(project.endDate);
-        project.employees = project.employees.length;
-      });
+      console.log(data, 'datA');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/redux/modules/projects/thunks.js:53]: We need this console.log?

File: src/Components/redux/modules/projects/thunks.js [hunk 6]
```diff
@@ -49,14 +50,9 @@ export const getProject = () => {
...
       dispatch(getProjectSuccess(data.data));
+      console.log(data.data[0], 'dos');
```
Comment (SuarezAlexisEzequiel) [inline src/Components/redux/modules/projects/thunks.js:55]: We need this console.log?

**Review by SuarezAlexisEzequiel — COMMENTED** [review]
Everything seems ok to me, only need to resolve the comments lefted.

File: src/Components/redux/modules/projects/thunks.js [hunk 7]
```diff
@@ -16,29 +16,30 @@ import {
...
+//     if (day < 10) {
+//       day = `0${day}`;
+//     }
+//     if (month < 10) {
+//       month = `0${month}`;
+//     }
+//     changedDate = `${month}-${day}-${year}`;
+//   }
+//   return changedDate;
+// };
```
Comment (marcomigliaro) [inline src/Components/redux/modules/projects/thunks.js:37]: Do we need this comments?

**Review by marcomigliaro — APPROVED** [review]

**Pull request merged**